src/main/java/com/softserve/academy/spaced/repetition/controller/CategoryController.java

```diff
@@ -56,6 +57,16 @@
         return new ResponseEntity<>(categories, HttpStatus.OK);
     }
 
+    @Auditable(action = AuditingAction.VIEW_TOP_CATEGORIES)
```

Comment:
Do you really need to audit each request for top categories? There will be too much entries into audit file.